include/openenclave/bits/eeid.h

```diff
@@ -0,0 +1,278 @@
+// Copyright (c) Open Enclave SDK contributors.
+// Licensed under the MIT License.
+
+/**
+ * @file eeid.h
+ *
+ * This file defines the EEID structure.
+ *
+ */
+
+#ifndef _OE_BITS_EEID_H
+#define _OE_BITS_EEID_H
+
+#include "properties.h"
+
+OE_EXTERNC_BEGIN
+
+#ifdef OE_WITH_EXPERIMENTAL_EEID
+
+// This is the public key corresponding to the private key OE_DEBUG_SIGN_KEY in
+// signkey.c/.h.
+static const uint8_t OE_DEBUG_PUBLIC_KEY[] = {
+    0xca, 0x9a, 0xd7, 0x33, 0x14, 0x48, 0x98, 0x0a, 0xa2, 0x88, 0x90,
+    0xce, 0x73, 0xe4, 0x33, 0x63, 0x83, 0x77, 0xf1, 0x79, 0xab, 0x44,
+    0x56, 0xb2, 0xfe, 0x23, 0x71, 0x93, 0x19, 0x3a, 0x8d, 0xa};
+
+/**
+ * Determine whether properties are those of a base image to be used with EEID
+ *
+ * @param[in] properties Properties of an SGX enclave.
+ *
+ * @returns Returns true if **properties** are consistent with those of an EEID
+ * base image.
+ *
+ */
+int is_eeid_base_image(const oe_sgx_enclave_properties_t* properties);
+
+/*
+**==============================================================================
+**
+** oe_eeid_t
+**
+**==============================================================================
+*/
+
+#define OE_EEID_VERSION (1)
+
+OE_PACK_BEGIN
+typedef struct _oe_eeid
+{
+    uint32_t version; /* Version of the oe_eeid_t structure */
```

Comment:
Can we make sure these are doxygen formatted for the public struct? (see `oe_enclave_setting_context_switchless_t` for example). Basically, add a leading doxygen comment for the struct as a whole, and the members should be `/** */` commented rather than `/* */`

Reply:
Done!